AudioBarTests/AudioBarViewTests.swift

```diff
@@ -0,0 +1,154 @@
+        expect(view.isPlayPauseButtonEnabled, false)
+        expect(view.areSeekButtonsHidden)
+        expect(view.playbackTime, "")
+        expect(view.isSeekBackButtonEnabled, false)
+        expect(view.isSeekForwardButtonEnabled, false)
+        expect(view.isLoadingIndicatorVisible, false)
+    }
+
+    func testViewWhenReadyToLoad() {
+        model = .readyToLoadURL(URL.arbitrary)
+        expect(view.playPauseButtonMode, .play)
+        expect(view.isPlayPauseButtonEnabled)
+        expect(view.areSeekButtonsHidden)
+        expect(view.playbackTime, "")
+        expect(view.isSeekBackButtonEnabled, false)
+        expect(view.isSeekForwardButtonEnabled, false)
+        expect(view.isLoadingIndicatorVisible, false)
+    }
+
+    func testViewWhenWaitingForPlayer() {
+        model = .waitingForPlayerToBecomeReadyToPlayURL(URL.arbitrary)
+        expect(view.playPauseButtonMode, .pause)
+        expect(view.isPlayPauseButtonEnabled)
+        expect(view.areSeekButtonsHidden)
+        expect(view.playbackTime, "")
+        expect(view.isSeekBackButtonEnabled, false)
+        expect(view.isSeekForwardButtonEnabled, false)
+        expect(view.isLoadingIndicatorVisible)
+    }
+
+    func testPlaybackTime1() {
+        model = .readyToPlay(.init(currentTime: nil))
+        expect(view.playbackTime, "")
+    }
+
+    func testPlaybackTime2() {
+        model = .readyToPlay(.init(duration: 1, currentTime: 0))
+        expect(view.playbackTime, "-0:01")
+    }
+
+    func testPlaybackTime3() {
+        model = .readyToPlay(.init(duration: 61, currentTime: 0))
+        expect(view.playbackTime, "-1:01")
+    }
+
+    func testPlaybackTime4() {
+        model = .readyToPlay(.init(duration: 60, currentTime: 20))
+        expect(view.playbackTime, "-0:40")
+    }
+
+    func testIsLoadingIndicatorVisible1() {
+        model = .readyToPlay(.init(isPlaying: true, currentTime: nil))
+        expect(view.isLoadingIndicatorVisible)
+    }
+
+    func testIsLoadingIndicatorVisible2() {
+        model = .readyToPlay(.init(isPlaying: false, currentTime: nil))
+        expect(view.isLoadingIndicatorVisible, false)
+    }
+
+    func testIsLoadingIndicatorVisible3() {
+        model = .readyToPlay(.init(isPlaying: true, currentTime: 0))
+        expect(view.isLoadingIndicatorVisible, false)
+    }
+
+    func testIsLoadingIndicatorVisible4() {
+        model = .readyToPlay(.init(isPlaying: false, currentTime: 0))
+        expect(view.isLoadingIndicatorVisible, false)
+    }
+
+    func testIsSeekBackButtonEnabled1() {
+        model = .readyToPlay(.init(currentTime: nil))
+        expect(view.isSeekBackButtonEnabled, false)
+    }
+
+    func testIsSeekBackButtonEnabled2() {
+        model = .readyToPlay(.init(duration: 60, currentTime: 0))
+        expect(view.isSeekBackButtonEnabled, false)
+    }
+
+    func testIsSeekBackButtonEnabled3() {
+        model = .readyToPlay(.init(duration: 60, currentTime: 1))
```

Comment:
In old tests we had currentTime: 0.01. I suppose there is no difference, but I'd suggest separate commits for test optimisations next time.